.github/workflows/build-sw-container.yml

```diff
@@ -11,7 +11,6 @@ on:
 jobs:
   make-date-tag:
     runs-on: ubuntu-latest
-    if: startsWith(github.repository, 'opensciencegrid/')
```

Comment:
do you want to keep this change, or were you only using it for testing?

Reply:
Yup, I wanted to keep this change because the build job depends on the date generation job and I think it's reasonable for the build GHA to run on forks. The only difference in forks vs upstream is that the build is pushed for upstream.